Edit reset service password section (#889)
timescale/docs: cloud/operations.md

</procedure>

### Reset service password
You can reset your service password from the `Operations` dashboard. This is the
password you use to connect to your database, not the password to your Timescale
Cloud account. To reset your Timescale Cloud password, navigate to the `Account`
page. 

> When you reset your service password, you are first prompted for your Timescale

Reviewer: ```suggestion
When you reset your service password, you are prompted for your Timescale
```

> Cloud password. Once you authenticate, you can create a new service password,

Reviewer: ```suggestion
Cloud password. When you have authenticated, you can create a new service password,
```

ask Timescale Cloud to auto-generate a password, or switch your authentication

> type between SCRAM and MD5.

Reviewer: Can we add a short description of SCRAM and MD5 here?

Reviewer 2: SCRAM = Salted Challenge Response Authentication Mechanism - https://en.wikipedia.org/wiki/Salted_Challenge_Response_Authentication_Mechanism
MD5 = Message Digest Algorithm  - https://en.wikipedia.org/wiki/MD5

Author: ```suggestion
type between SCRAM and MD5.

SCRAM (Salted Challenge Response Authentication Mechanism) and MD5 (Message Digest
Algorithm 5) are cryptographic authentication mechanisms. Timescale Cloud uses SCRAM
by default. It is more secure and strongly recommended. The MD5 option is provided
for compatibility with older clients.
```

How about this?

Reviewer 2: 💯
